src/test/librados/misc.cc

```diff
@@ -510,6 +510,29 @@ TEST_F(LibRadosMiscPP, AssertExistsPP) {
   ASSERT_EQ(-EEXIST, ioctx.create("asdffoo", true));
 }
 
+TEST_F(LibRadosMiscPP, AssertVersionPP) {
+  char buf[64];
+  memset(buf, 0xcc, sizeof(buf));
+  bufferlist bl;
+  bl.append(buf, sizeof(buf));
+
+  ASSERT_EQ(0, ioctx.create("asdfbar", true));
+  ASSERT_EQ(0, ioctx.write_full("asdfbar", bl));
```

Comment:
not that it really matters, but write_full implicitly creates so the create op above could be dropped